lib/geoblacklight/dataingest.rb

```diff
@@ -84,8 +82,11 @@ def is_valid_url(urlstring)
 
 
   def validaterecord(content)
+    if content.length === 0
+      return "Row does not contain record."
+    end
     result = ""
-    fields = ["uuid","dc:identifier", "dc:rights", "dct:provenance", "dct:references",
```

Comment:
when you remove uuid. the index in this line also changes too. 

if item[1].nil? and ![0, 8, 11, 12, 15, 16, 20, 22].include? i